boards/arm/sama5/giant-board/src/sam_nandflash.c

```diff
@@ -0,0 +1,212 @@
+/****************************************************************************
```

Comment:
This should be removed.  There is NAND on the Giant Board.  It just confuses things cloning files which are not appropriate for the board.  It is a bad practice.  Please remove.

Reply:
@patacongo will fix, sorry.